src/api/nft.ts

```diff
@@ -51,6 +63,10 @@ export interface BaseNft {
 /**
  * Alchemy representation of an NFT.
  *
+ * The BaseNft object does not hold any metadata information and only contains
+ * the NFT contract and token ID. The Nft object additionally contains the NFT
```

Comment:
can we wrap The references to the objects with `{@link}` to make it easier to read?